lib/connect.js

```diff
@@ -129,8 +135,10 @@ let didPrintDscpTaggingWarning = false;
  *   room.localParticipants.trackPublications.forEach(function(publication) {
  *     console.log('The LocalTrack "' + publication.trackName + '" was successfully published');
  *   });
+ * }).catch(error => {
+ *   console.log('Could not connect to the room: ' + error.message);
```

Comment:
@makarandp0 Nitpick:
```suggestion
 *   console.log('Could not connect to the Room:', error.message);
```